tests/state_test_util.go

```diff
@@ -318,6 +318,8 @@ func (t *StateTest) RunNoVerify(subtest StateSubtest, vmconfig vm.Config, snapsh
 		rnd := common.BigToHash(t.json.Env.Random)
 		context.Random = &rnd
 		context.Difficulty = big.NewInt(0)
+	} else {
+		context.Difficulty = t.json.Env.Difficulty
```

Comment:
I don't think -- off the top of my head -- this is not necessary; there may be supporting logic in the `.genesis(fork)` call that sets this too.  `Difficulty` field, touchy touchy.